### DISCUSSION TITLE
html reports - curve instead of histogram

### DISCUSSION THREAD

**Discussion by torebo** [artilleryio/artillery#1844] [Artillery]
Hello everybody,
Using Artillery CLI, version 2.0.0.28, the html reports show all graphes as histograms. Is there a way, an option, to change the type of graphe shown? For example, I would prefer seeing curves than histograms.  
Thank you
Best regards

**Comment by swiss2487** [comment]
I would like that also :-) as the reporting is important and I was hoping for something like the gatling report
